### PR TITLE
fix: update polygon parts fields

### PR DESCRIPTION
**Pull request by RonitKissis**
updated polygon parts fields.
id -> uuid - unique record id
partId -> uuid - related part id
recordId -> catalogId
id -> sourceId - external id
name -> sourceName - external name
updatedInVersion -> productVersion

fixed typo in producType -> productType

### CODE REVIEW

File: docs/MapColonies/Raster/services/PolygonParts/profile_v1_0.md [hunk 1]
```diff
@@ -16,13 +16,14 @@ tags:
 | **Polygon Part Queryable <br/> Attribute Name** | **Type** | **Description** | 
 | ----------- | ----------- | ----------- | 
 | geometry | GeoJson (geometry (Polygon,4326)) | The feature's geometry |
-| partId | integer | Internal unique part identifier (1-2147483647) |
-| recordId | UUID | Mapcolonies reaster catalog identifier for the layer |
+| id | UUID | Internal unique identifier |
+| partId | UUID | Part identifier |
```
Comment (shimoncohen) [inline docs/MapColonies/Raster/services/PolygonParts/profile_v1_0.md]: Please remove this. Remove from response too.
This isn't relevant for the end users.

File: docs/MapColonies/Raster/services/PolygonParts/README.md [hunk 2]
```diff
@@ -469,13 +478,14 @@ Now that we hold the structure of the `orthophoto_mosaic_base_ORTHOPHOTO_MOSAIC_
...
+        "catalogId": "a1b6dd1c-c77c-42e6-a00a-306b67bcdebf",
         "productId": "ORTHOPHOTO_MOSAIC_BASE",
-        "producType": "OrthophotoBest",
-        "id": "MAS_6_ORT_247568-26.0",
-        "name": "O_south_w84geo_Tiff_10cm",
-        "updatedInVersion": "2",
+        "productType": "OrthophotoBest",
+        "sourceId": "MAS_6_ORT_247568-26.0",
+        "sourceName": "O_south_w84geo_Tiff_10cm",
+        "productVersion": "2.0",
```
Comment (shimoncohen) [inline docs/MapColonies/Raster/services/PolygonParts/README.md:478]: Was this intentional? Do all of the versions need to be fixed to this format?

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: docs/MapColonies/Raster/services/PolygonParts/README.md (hunk 2 above)
Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:478]: I think productVersion must be X.X (with up to 2 numbers after the dot) 
but in the new ingestion trigger we set the version to 1.0 and increase it by 1 in update
i will change all the versions to have a .0

Comment (RonitKissis) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:478]: the point is that they will know that version is a string

**Review by shimoncohen — APPROVED** [review]

File: docs/MapColonies/Raster/services/PolygonParts/README.md (hunk 2 above)
Comment (shimoncohen) [reply docs/MapColonies/Raster/services/PolygonParts/README.md:478]: Great, thank you!

**Review by CL-SHLOMIKONCHA — APPROVED** [review]

**Pull request merged**